AGENTS.md

```
@@ -0,0 +1,8 @@
+- Always run `uv run black src; uv run ruff check src --fix; uv run pyrefly check src; uv run ty check src` after making changes.
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)